mrmustard/utils/xptensor.py

```diff
@@ -25,7 +25,7 @@
 class XPTensor(ABC):
     r"""A representation of Matrices and Vectors in phase space.
 
-    Tensors in phase space have a (2n, 2n) or (2n,) shape where n is the number of modes.
+    Tensors in phase space have a ``(2n, 2n)`` or ``(2n,)`` shape where n is the number of modes.
     There are two main orderings:
```

Comment:
```suggestion
    Tensors in phase space have a ``(2n, 2n)`` or ``(2n,)`` shape where n is the number of modes.

    There are two main orderings:
```

Reply:
This docstring would benefit from some restructuring (e.g., empty lines between paragraphs along with switching to double backticks for code). There are also quite a few missing docstrings.